+++ upgrade.sh  2021-09-24 20:02:41.979461348 +0000
@@ -56,6 +56,9 @@
   echo "Skipping local dependencies (local_requirements.txt not found)"
 fi

+# patch
+sed -i 's/h = hashlib.md5()/h = hashlib.md5(usedforsecurity=False)/1' /opt/netbox/venv/lib64/python3.6/site-packages/django/db/backends/utils.py
+sed -i 's/= hashlib.md5()/= hashlib.md5(usedforsecurity=False)/1' /opt/netbox/venv/lib64/python3.6/site-packages/cacheops/utils.py
 # Apply any database migrations
 COMMAND="python3 netbox/manage.py migrate"
 echo "Applying database migrations ($COMMAND)..."
